### ISSUE TITLE
NVDA does not read alerts from YouTube when opened in Edge

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#6920]
I've noticed that NVDA does not read alerts from YouTube, if the site is opened with Edge. Such alerts are the subtitles, the appearance of advertisements, etc. Actually, NVDA seams not to announce alerts from other sites opened in Edge as well. A good example for that is
http://beta.speedtest.net
When you activate the "Go" link, NVDA should announce the status of the speed test via alerts, but it does not. There are such alerts, because NVDA announces them if Firefox or Internet Explorer are used for the purpose.

Those two pages seam to be only examples. If I'm not mistaking, NVDA does not read alerts from websites in Edge at all.

I've restarted with add-ons disabled, but the issue persisted.

Steps to reproduce it:
1. Open Microsoft Edge.
2. Go to YouTube and open a video with added subtitles. NVDA should read aloud the subtitles and the appearance of advertisements, but it does not.
3. Go to the HTML5 version of the Speed Test by Ookla (http://beta.speedtest.net) and activate the "Go" link to start the test. NVDA should announce the status of the speed test procedure with alerts, but it does not.

Test environment:
- Operating system: Windows 10 Pro (build 14393.693), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- Microsoft Edge version: 11.00.14393.82 (Microsoft Edge 38.14393.0.0, Microsoft EdgeHTML 14.14393).
- NVDA version: next-13899,af44c1d4, in bulgarian.
- Processor: Intel Core i5-2320 at 3.00GHz.
- RAM Memory: 8.00GB.
- Graphics: Palit GeForce GT 630, 1024MB dedicated memory + 3071MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC662 at Intel Cougar Point PCH - High Definition Audio Controller.

**Comment by Adriani90** [comment]
@k-kolev1985 I assume allerts are read properly in Edge Chromium 81 or newer. Right?

**Comment by k-kolev1985** [comment]
@Adriani90 Yes, as far as I could test, alerts are read properly in Chromium-based Microsoft Edge.

**Comment by Adriani90** [comment]
Thanks for reporting back, then I close this as works for me since Edge Chromium will be distrobuted to most users via Windows 10 20 H1 update.
